packages/protocol/contracts/L1/provers/Guardians.sol

```diff
@@ -41,14 +41,7 @@ abstract contract Guardians is EssentialContract {
     /// @notice Set the set of guardians
     /// @param _newGuardians The new set of guardians
     /// @param _minGuardians The minimum required to sign
-    function setGuardians(
-        address[] memory _newGuardians,
-        uint8 _minGuardians
-    )
-        external
-        onlyOwner
-        nonReentrant
```

Comment:
this fixes the `pnpm deploy:foundry` failure, not sure why.

Reply:
The reason is this:
```
 Error: 
Transaction Failure: 0x63379e0e6ddddf647171f9fcde954b7a10c04896edc09938851840a7bbeaad7d
eth_chainId
EIP-3855 is not supported in one or more of the RPCs used.
```

So anvil i guess does not have dencun upgrade yet!

Reply:
oh, that's why!